.github/workflows/ci-cd.yml

```
@@ -30,8 +30,82 @@ jobs:
+            echo "Backend is up and running!"
+          else
+            echo "Backend failed to start"
+            exit 1
+          fi
+
+      - name: Wait for backend
+        run: |
+          echo "Waiting for backend to start..."
+          max_attempts=12
+          attempts=0
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)